client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -53,7 +53,13 @@ class DeployedModel(object):
     Examples
     --------
     .. code-block:: python
+       :emphasize-lines: 3,4
+
+        # Preferred method for instantiating an object.
+        deployed_model_x = verta.Client().get_or_create_endpoint('endpoint_name').get_deployed_model()
+        deployed_model_x.predict(['here is a prediction'])
```

Comment:
For some consistency with the rest of our docs:

```suggestion
        endpoint = client.get_or_create_endpoint('endpoint_name')
        deployed_model = endpoint.get_deployed_model()
        deployed_model.predict(['here is a prediction'])
```